lib/qx/tool/cli/commands/contrib/Install.js

```diff
@@ -88,28 +88,31 @@ qx.Class.define("qx.tool.cli.commands.contrib.Install", {
         repos_cache = this.getCache().repos;
       }
 
-      if (!argv.repository) {
+      if (!argv.contrib_uri) {
```

Comment:
does this break existing contrib.json files?  I'm not against the change, just wondering if there's any steps that need to be taken

Reply:
No, I made an effort to make this backwards-compatible. The `contrib.json/libraries[]` maps get an additional `uri` property but stay the same otherwise. `qx contrib install` without arguments works just as before.